NOTICE.txt

```diff
@@ -65,12 +65,7 @@ by Adobe Systems Incorporated:
 Modified redistributions
 ========================
 
-This product contains a modified part of Brave, distributed by Zipkin.io:
-
-  * License: licenses/LICENSE.brave.al20.txt (Apache License v2.0)
-  * Homepage: https://github.com/openzipkin/brave/
```

Comment:
We do not have any code forked from Brave anymore thanks to the recent update by @anuraaga